### ISSUE TITLE
Not Showing All Episodes

### ISSUE THREAD

**Issue by TONYSOAD05** [jhassan8/crunchyroll-tizen#40]
**Describe the bug**
I have a premium account, so when I tried to see Attack on Titan I noticed that I can only see the 1st season in the App. I went back to the Web in my PC and reproduced a chapter of season 4, so it will stay on the "continue viewing" section. Once I reopened the App I can see that episode there, so I clicked in continue viewing it but it stays on an infinite black screen. 

The same happened with Naruto Shippudden and One Piece.

**To Reproduce**
1. Go to Crunchyroll App an try to reproduce Attack on Titan seasons 2-4.
2. There are no episodes there.
3. See error.

TV Models Tested:
QN55QN85AAFXZX
QN55Q60TAFXZX

**Comment by TONYSOAD05** [comment]
Hello, I have a couple of updates:

1. I reinstalled the whole thing again since scratch, I deleted the App and started a new installation process. On my previous process once I finished and the App was installed, immediately went to Apps Menu and disabled the "Developer Mode" this time I left it.

2. Once I click Login I noticed that now this time my user has an "orange crown", so looks like the first time it did not recognized that my account was premium; now it does. So, I checked the previous titles mentioned and now are working properly.

3. My assumption is that when I disabled the "Developer Mode" right after the installation I messed something, or maybe just the App did not recognized my premium account in the first time and all the premium content was immediately blocked or deleted. Maybe adding a feature to close session and login again could fix that.

Anyway, hopefully this information is useful to you; for the moment everything looks perfect :)

I reinstalled and followed the above process in my two TV's and its working properly:

QN55QN85AAFXZX
QN55Q60TAFXZX
